airlock/templates/file_browser/request/back_link_or_title.html

```diff
@@ -0,0 +1,9 @@
+<div hx-swap-oob="innerHTML:.header__actions">
+  {% if is_request_root %}
+    <h2 class="font-semibold tracking-tight text-slate-900">Request overview</h2>
+  {% else %}
+    <a class="header__link header__nav-link" href="{{ release_request.get_url }}" id="request-home-button">
+      &larr; Request overview
+    </a>
+  {% endif %}
+</div>
```

Comment:
There is a [ticket](#718) for adding the group name into the header that will also need an hx-swap-oob when users navigate with the tree. Is it worth thinking about how to do that at the same time? We don't want to end up with lots of bits of oob swaps, so would it make sense to replace the whole header rather than just the title/back link?

Reply:
I'm also looking at updates to the action-required alert (https://github.com/opensafely-core/airlock/issues/797), which I think also want an oob swap for, so it can have a link to the overview page but not if you're already on that page. So that's basically everything in `templates/file_browser/request/index.html`.

Reply:
Good idea. I'll see if I can rework it a bit.

Reply:
The whole "header" now updates whenever the file browser contents change via htmx request - but only for the file browser for "requests".

I considered renaming the "content" block to something more meaningful e.g. "header", but that pattern appears in lots of places so I just left it as is.